wmn-data.json

```diff
@@ -2004,6 +2004,16 @@
         "known" : ["tommy", "cderalow"],
         "cat" : "health"
        },
+        {
+        "name" : "GeeksForGeeks",
+        "uri_check" : "https://www.geeksforgeeks.org/user/{account}",
+        "e_code" : 200,
+        "e_string" : "|  GeeksforGeeks Profile</title>",
+        "m_code" : 404,

```

Comment:
I get a 307 code that redirects my browser to the 404.html page.

Reply:
I used Python requests and tried to get an invalid username, and it was returned with 404 as the final response code. If I do `response.history` it does give me the intermediate 307. `wget` has the same behaviour. 
I am not sure which status code is actually expected.


![image](https://github.com/user-attachments/assets/bf32bd5b-fda2-4aeb-9444-10425878fe04)

Reply:
For the project, we use the first response code (307) instead of the final one.